wvc-ourwork-nextjs/components/StatisticCardGrid/StatisticCardGrid.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../styles/lumen-tokens';
+
+.statistic-card {
+  background-color: $white;
+  border-radius: 2px;
+  box-shadow: 2px 2px 16px rgba(0, 0, 0, 0.05);
```

Comment:
Let's continue using rem moving forward!

Reply:
From my experience box-shadow is more often written with pixels due to how small the values are. Same with something like border-radius. (lumen token border radius also written in pixel) I'll leave this for now but we can change later if needed